src/Interfaces/AbstractHyperrectangle.jl

```diff
@@ -617,3 +617,21 @@ function volume(H::AbstractHyperrectangle{N}) where {N<:Real}
     vol = mapreduce(x -> 2x, *, radius_hyperrectangle(H))
     return vol
 end
+
+"""
+    center(H::AbstractHyperrectangle{N}, i::Int) where {N<:Real}
+
+Return the center along a given dimension of a hyperrectangular set.
+
+### Input
+
+- `H` -- hyperrectangular set
+- `i` -- dimension of interest
+
+### Output
+
+The center along a given dimension of the hyperrectangular set.
+"""
+function center(H::AbstractHyperrectangle{N}, i::Int) where {N<:Real}
+    return H.center[i]
```

Comment:
This doesn't work, because one can't guarantee that *any* implementation of an `AbstractHyperrectangle` contains a field named `center`. For example, think about an `Interval`, which does not have a `center` field, although one can construct its center by calling the `center` function.

Reply:
Since `center` is a required field for any implementation of the `AbstractHyperrectangle` interface, you can fix this by returning `center(H)[i]`.